addons/calendar/static/src/js/calendar_renderer.js

```diff
@@ -36,7 +36,7 @@ const AttendeeCalendarPopover = CalendarPopover.extend({
      * @return {boolean}
      */
     isCurrentPartnerAttendee() {
-        return this.event.extendedProps.record.partner_ids.includes(session.partner_id);
+        return this.event.extendedProps.record.attendee_status;
```

Comment:
Nope, this method aims to see if the current partner is part of the list of partners. it returns a boolean (see the docstring)
Your change modifies the return type into a string and it returns the state.
This is completely different and it does not fulfilled the desired purpose of the function.

Reply:
Yes but the problem is that this function asks if the current partner is attendee, however the current check does not fulfill that purpose, since someone not attending can have his session.partner_id inside partner_ids

But I agree, I will change it to return Boolean(this.event.extendedProps.record.attendee_status); so that it will be true if there is one of the 4 possible status, and false when there is no status